kubeconform/analyse.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+#
+# This script iterates over all kustomize overlays under specified
+# directory, renders the resulting manifests and performs a kubeconform check on them
+import subprocess
+import tempfile
+import argparse
+import os.path
+import sys
+from pathlib import Path
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('analyse_dir', nargs='?', default='.')
+parser.add_argument('--schema-location', action='append')
+
+args = parser.parse_args()
+
+kubeconform_cache = os.environ.get('KUBECONFORM_CACHE')
+if not kubeconform_cache:
+    kubeconform_cache = tempfile.mkdtemp()
+
+
+def analyse_overlay(overlay_dir):
+    print(f"🧐 {overlay_dir}: \t", end='', flush=True)
+
+    extra_args = []
+    if args.schema_location:
+        for location in args.schema_location:
+            extra_args.extend(['-schema-location', location])
+
+    p1 = subprocess.Popen(["kubectl", "kustomize", overlay_dir], stdout=subprocess.PIPE)
+    p2 = subprocess.Popen(["kubeconform", "--summary", "--exit-on-error", "--cache", kubeconform_cache] + extra_args,
```

Comment:
according [the docs](https://github.com/yannh/kubeconform) all the flags should have a single dash, but I guess it accepts both seeing that before they were mixed...

I'd rather use the same style everywhere, so
```suggestion
    p2 = subprocess.Popen(["kubeconform", "-summary", "-exit-on-error", "-cache", kubeconform_cache] + extra_args,
```